src/dev/precommit_hook/casing_check_config.js

```diff
@@ -106,7 +106,6 @@ export const IGNORE_DIRECTORY_GLOBS = [
 export const TEMPORARILY_IGNORED_PATHS = [
   'src/legacy/core_plugins/console/public/src/directives/helpExample.txt',
   'src/legacy/core_plugins/console/public/src/sense_editor/theme-sense-dark.js',
-  'src/legacy/core_plugins/kibana/public/assets/play-circle.svg',
```

Comment:
@elastic/kibana-operations this file was removed, that's why I also removed this line